### ISSUE TITLE
 Error executing python file Train_YOLO.py

### ISSUE THREAD

**Issue by brufeer** [AntonMu/TrainYourOwnYOLO#83]
![Captura de tela de 2020-02-12 16-58-48](https://user-images.githubusercontent.com/50056231/74437906-f858ef80-4e36-11ea-90d3-d5f88cb5d250.png)

I'm using the virtual environment.

**Comment by brufeer** [comment]
hi, i'm testing your YOLO training, i run it in a virtual environment and in the last lines the error "attribute error".

**Comment by brufeer** [comment]
I managed to run or Train_YOLO.py file and it returned this error : 
"...
 .cc:533] shape_optimizer failed: Invalid argument: Subshape must have computed start >= end since stride is negative, but is 0 and 2 (computed from start 0 and end 9223372036854775807 over shape with rank 2 and stride-1)
2020-02-13 14:34:45.014097: E tensorflow/core/grappler/optimizers/meta_optimizer.cc:533] remapper failed: Invalid argument: Subshape must have computed start >= end since stride is negative, but is 0 and 2 (computed from start 0 and end 9223372036854775807 over shape with rank 2 and stride-1)
2020-02-13 14:34:45.843150: W tensorflow/core/framework/cpu_allocator_impl.cc:81] Allocation of 708837376 exceeds 10% of system memory.
2020-02-13 14:34:47.184423: W tensorflow/core/framework/cpu_allocator_impl.cc:81] Allocation of 712249344 exceeds 10% of system memory.
2020-02-13 14:34:47.731849: W tensorflow/core/framework/cpu_allocator_impl.cc:81] Allocation of 354418688 exceeds 10% of system memory.
2020-02-13 14:34:48.072292: W tensorflow/core/framework/cpu_allocator_impl.cc:81] Allocation of 354418688 exceeds 10% of system memory.
2020-02-13 14:34:48.503920: W tensorflow/core/framework/cpu_allocator_impl.cc:81] Allocation of 177209344 exceeds 10% of system memory.
2/5 [===========>..................] - ETA: 1:10 - loss: 6342.5593Morto  "
